Source/ACE/Managers/GuidManager.cs

```diff
@@ -45,6 +53,16 @@ public static uint NewPlayerGuid()
         }
 
         /// <summary>
+        /// Returns New Admin Guid
+        /// </summary>
+        /// <returns></returns>
+        public static uint NewAdminGuid()
```

Comment:
in the sense of spawning things in world, it works, but you're absolutely correct it needs to be looked over and fleshed out so that it can be used to generate correct guids for new items to be saved in ace_shard, and persist so that it doesn't reuse incorrectly. gotta get everything on the same page first and then we have a fighting chance :)